Source/WebCore/platform/graphics/harfbuzz/ComplexTextControllerHarfBuzz.cpp

```diff
@@ -195,9 +195,9 @@ static Vector<hb_feature_t, 4> fontFeatures(const FontCascade& font, const FontP
 
     // 3. Font features implied by the value of the ‘font-variant’ property, the related ‘font-variant’
     //    subproperties and any other CSS property that uses OpenType features.
-    for (auto& feature : computeFeatureSettingsFromVariants(font.fontDescription().variantSettings()))
+    for (auto& feature : computeFeatureSettingsFromVariants(font.fontDescription().variantSettings(), { }))
```

Comment:
Might want to add a FIXME or something, with a bug URL

Reply:
Added